.github/workflows/ci.yml

```diff
@@ -228,14 +228,22 @@ jobs:
         if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
       - name: Build latest earthly using released earthly
         run: earthly --use-inline-cache +for-linux
-      - name: Push and Pull Images (not a fork)
+      - name: Push and Pull Cloud Images (not a fork)
         run: |
           export EARTHLY_BUILD_ARGS="DOCKERHUB_USER_SECRET=+secrets/earthly-technologies/dockerhub/user,DOCKERHUB_TOKEN_SECRET=+secrets/earthly-technologies/dockerhub/token"
           ./build/linux/amd64/earthly --ci -P ./examples/tests/cloud-push-pull+all
         if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
-      - name: Push and Pull Images (fork)
+      - name: Push and Pull Cloud Images (fork)
         run: ./build/linux/amd64/earthly --ci -P ./examples/tests/cloud-push-pull+all
         if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name != github.repository
+      - name: Push Images after RUN --push (not a fork)
+        run: |
+          export EARTHLY_BUILD_ARGS="DOCKERHUB_USER_SECRET=+secrets/earthly-technologies/dockerhub/user,DOCKERHUB_TOKEN_SECRET=+secrets/earthly-technologies/dockerhub/token"
+          ./examples/tests/push-images/test.sh
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+      - name: Push Images after RUN --push (fork)
+        run: ./examples/tests/push-images/test.sh
```

Comment:
The fork version is not necessary because this test is disabled for fork PR (it requires credentials).